src/com/goide/runconfig/testing/coverage/GoCoverageMerger.java

```diff
@@ -0,0 +1,60 @@
+package com.goide.runconfig.testing.coverage;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VfsUtil;
+import com.intellij.openapi.vfs.VirtualFile;
+
+import java.io.*;
+
+public class GoCoverageMerger {
+  protected static final Logger LOG = Logger.getInstance("#com.goide.runconfig.testing.coverage.GoCoverageMerger");
+
+  public static void MergeCoverage(String fromPath, String toFile) {
+    try (
+        FileWriter stream = new FileWriter(toFile, false);
+        BufferedWriter writer = new BufferedWriter(stream)
+    ) {
+      writer.write("mode: set");
+      writer.newLine();
+      VirtualFile fromFile = LocalFileSystem.getInstance().findFileByPath(fromPath);
+      VfsUtil.markDirtyAndRefresh(false, true, true, fromFile);
```

Comment:
Merger should not know anything about VFS refreshing, it should just merge – that's it. 
It's a client who knows that the files were appeared a moment ago and need to be refreshed.